src/qz/printer/status/StatusSession.java

```diff
@@ -21,7 +21,7 @@ public void statusChanged (PrinterStatus printerStatus) {
 
     private StreamEvent createStatusStream(PrinterStatus status) {
         return new StreamEvent(StreamEvent.Stream.PRINTER, StreamEvent.Type.ACTION)
-                .withData("printerName", status.issuingPrinterName)
```

Comment:
Please keep `issuingPrinterName` and add `issuingPrinterDescription` as an additional field.